error.js

```diff
@@ -0,0 +1,10 @@
+exports.handlePsqlErrors = (err,req,res,next) =>{
+
+}
+
+exports.handle404Errors = (err,req,res,next) =>{
```

Comment:
this function is called handle 404 errors but it's sending back a 500 status code...